Makefile

```diff
@@ -370,7 +377,6 @@ build-volumes:
 	$(SILENT)mkdir -p $(CURDIR)/linux-gocache
 	$(SILENT)docker volume inspect $(GOPATH_VOLUME_NAME) >/dev/null 2>&1 || docker volume create $(GOPATH_VOLUME_NAME)
 	$(SILENT)docker volume inspect $(GOCACHE_VOLUME_NAME) >/dev/null 2>&1 || docker volume create $(GOCACHE_VOLUME_NAME)
-	$(SILENT)docker run --rm $(LOCAL_VOLUME_ARGS) $(BUILD_IMAGE) chown -R "$(shell id -u)" /linux-gocache /go
```

Comment:
This command was here to allow transparent change from root-owned directories (which would typically be the case for anyone who ever ran a dockerized build before `--user ...` flags were added) to regular-user-owned directories. And AFAICT this works for non-podman users.

If this does not work for podman for some reason, and some manual intervention is required, then perhaps you can make this line conditional to not break the user experience for non-podman users? :thinking:

Reply:
@porridge , I reverted this change and added `$(DOCKER_USER)` there, and to a line in `make/protogen.mk`.

Reply:
update: `chown` doesn't work for me actually, I'll need to do it conditionally.
```
chown: changing ownership of '/go/pkg/mod/gorm.io/driver/postgres@v1.3.7': Operation not permitted
```

Reply:
I'm not sure that the idea of changing ownership of users' GOPATH is a good one. If we corrupted it once, let the user fix it properly.
I suggest removing this `chown`.

Reply:
I also felt a bit uneasy about it, but it seems to have done the trick for everyone so far except for `podman` users and those who happened to run `make` using `sudo`.
How about we make it conditional on `!(,$(findstring podman,$(DOCKER_VERSION)))` for now, and delete in a couple of weeks, once the ownership is fixed for vast majority of engineers?

Reply:
Not a big fan of it, I'd prefer engineers to suffer. I added an echo nevertheless.